content/docs/user-guide/basic-concepts/file-link.md

```diff
@@ -0,0 +1,9 @@
+---
+name: File Linking
+match: [linked]
+tooltip: >-
+  A way to have a file appear in multiple different folders without occupying
+  more physical space on the storage disk. This is both fast and economical. See
```

Comment:
> A way to have a file appear in multiple different folders

This is not a DVC concept. Idk if we want to explain file linking in general.

I do agree this tooltip could be useful but it should prob. be a very summarized version of https://dvc.org/doc/user-guide/large-dataset-optimization

Can we extract this change (and https://github.com/iterative/dvc.org/pull/2359#discussion_r610314439) to it's own PR please, so it can be applied in more places and reviewed without getting lost in 100 misc copy edits? Esp. since you're taking over #550, @casperdcl . 🙏

Reply:
> This is not a DVC concept.

No, but I wasn't aware that `<abbr>`s were used exclusively for DVC concepts. I understood they are used as tooltips in place of footnotes and thus replaced a very ugly manual footnote with this tooltip.

> Idk if we want to explain file linking in general.

Clearly needs a tooltip/footnote/aside as evidenced by the footnote before this change.

> I do agree this tooltip could be useful but it should prob. be a very summarized version of [dvc.org/doc/user-guide/large-dataset-optimization](https://dvc.org/doc/user-guide/large-dataset-optimization)

That is exactly what it is. It links to exactly that page. Other `<abbr>`s also summarise & link to appropriate pages.

> Can we extract this change (and [#2359 (comment)](https://github.com/iterative/dvc.org/pull/2359#discussion_r610314439)) to it's own PR please, so it can be applied in more places and reviewed without getting lost in 100 misc copy edits?

If we're going to keep it isn't it easier to merge now and immediately open an issue to continue review?

The point of this PR was to fix anything to do with *Getting Started* but we can change it to exclusively be copyedits only if you prefer.

Reply:
> I understood they are used as tooltips in place of footnotes and thus replaced a very ugly manual footnote with this tooltip

This is a good question. I think we can be somewhat flexible: not all tooltips will become full-blown basic concepts docs (i.e. the ones included in #550). But still they're not as broad as to replace any note.

In this case file linking is key for some DVC features so a tooltip can help, let's just not explain what file linking is? We can just focus on which kinds we support, related commands, etc.

> Clearly needs a tooltip/footnote/aside as evidenced by the footnote before this change.

The previous note didn't explain what file linking is. https://dvc.org/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache does but that doc is about file linking so it feels more natural to give some context there (we could replace the short explanation for a link though). Explaining directly in the GS or in a tooltip that will pop up in different contexts doesn't seem necessary.

> isn't it easier to merge now
we can change it to exclusively be copyedits only if you prefer

We can't merge since we have so many discussions. That's why I want to split this. So we can start merging something. Please see the specific plan I suggested as to how to proceed. Thanks